addons/primitives/prim_opengl.c

```diff
@@ -671,8 +671,9 @@ int _al_draw_indexed_buffer_opengl(ALLEGRO_BITMAP* target, ALLEGRO_BITMAP* textu
 #endif
 }
 
+#include <stdio.h>
```

Comment:
We should add this to our internal headers :)

Reply:
Ahh, yeah, at least I remembered to remove my printfs themselves!